packages/client/src/App.tsx

```diff
@@ -10,7 +10,7 @@ function App() {
       <Route path="/" exact>
         <Lobby />
       </Route>
-      <Route path="/game/:roomName">
+      <Route path="/game/:roomId">
```

Comment:
just thinking shall we keep roomName here and leave the ability to change it to the player now that we rely on id everywhere else?